nexus/tests/integration_tests/schema.rs

```diff
@@ -456,29 +470,50 @@ const REFERENTIAL_CONSTRAINTS: [&'static str; 8] = [
 const VIEWS: [&'static str; 4] =
     ["table_catalog", "table_schema", "table_name", "view_definition"];
 
-const STATISTICS: [&'static str; 8] = [
+const STATISTICS: [&'static str; 11] = [
     "table_catalog",
     "table_schema",
     "table_name",
     "non_unique",
     "index_schema",
     "index_name",
+    "seq_in_index",
     "column_name",
     "direction",
+    "storing",
+    "implicit",
 ];
 
+const PG_INDEXES: [&'static str; 5] =
+    ["schemaname", "tablename", "indexname", "tablespace", "indexdef"];
+
 const TABLES: [&'static str; 4] =
     ["table_catalog", "table_schema", "table_name", "table_type"];
 
+const TABLE_CONSTRAINTS: [&'static str; 9] = [
+    "constraint_catalog",
+    "constraint_schema",
+    "constraint_name",
+    "table_catalog",
+    "table_schema",
+    "table_name",
+    "constraint_type",
+    "is_deferrable",
+    "initially_deferred",
+];
```

Comment:
Similar to `CONSTRAINT_COLUMN_USAGE`, this wasn't necessary here, but it seems useful to ensure that it doesn't get out-of-sync regardless.

https://www.cockroachlabs.com/docs/stable/information-schema#table_constraints